GpuMiner/MinerEngine/CLMiner.cpp

```diff
@@ -462,19 +473,33 @@ bool CLMiner::Initialize()
 
         // create miner OpenCL program
         cl::Program::Sources sources { { _kernelCode.data(), _kernelCode.size() } };
-        cl::Program program(_context, sources);
+        cl::Program program(_context, sources);
         try
-        {
+        {
             program.build({ device }, options);
             cllog << "Build info:" << program.getBuildInfo<CL_PROGRAM_BUILD_LOG>(device);
         }
         catch(cl::Error const&)
         {
             cwarn << "Build info:" << program.getBuildInfo<CL_PROGRAM_BUILD_LOG>(device);
             return false;
-        }
-
-        _searchKernel = cl::Kernel(program, "search_nonce");
+        }
+
+

```

Comment:
Two empty lines :)

Reply:
Aha, yes, no need.
Btw, GpuMiner used Windows CR\LF, but for xdag it use Unix LF. Each time modify two project, have to change the preference of xcode.
Maybe we can solve the CR\LF in future.